datafusion/core/src/datasource/listing/mod.rs

```diff
@@ -88,11 +88,12 @@ impl std::fmt::Display for PartitionedFile {
     }
 }
 
-/// Helper method to fetch the file size and date at given path and create a `FileMeta`
-pub fn local_unpartitioned_file(file: String) -> PartitionedFile {
-    PartitionedFile {
-        file_meta: local::local_unpartitioned_file(file),
-        partition_values: vec![],
-        range: None,
+impl From<FileMeta> for PartitionedFile {
```

Comment:
This not only makes for reduced verbosity, but avoids a confusing situation where there are two local_unpartitioned_file, one which returns FileMeta and one which returns PartitionedFile